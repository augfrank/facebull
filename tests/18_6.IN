M0   C0   C3  2949
M1   C0   C4  2386
M2   C0   C6  2709
M3   C0   C9  2816
M4   C0   C13  2807
M5   C0   C14  2990
M6   C0   C15  2322
M7   C0   C17  2053
M8   C1   C2  2567
M9   C1   C3  2138
M10   C1   C4  2187
M11   C1   C5  2787
M12   C1   C6  2979
M13   C1   C7  2386
M14   C1   C8  2597
M15   C1   C10  2726
M16   C1   C12  2925
M17   C1   C13  2486
M18   C1   C14  2135
M19   C1   C16  2189
M20   C1   C17  2253
M21   C2   C0  2442
M22   C2   C3  2069
M23   C2   C5  2394
M24   C2   C6  2082
M25   C2   C8  2505
M26   C2   C10  2949
M27   C2   C11  2797
M28   C2   C13  2921
M29   C2   C15  2270
M30   C2   C16  2587
M31   C2   C17  2693
M32   C3   C0  2251
M33   C3   C1  2204
M34   C3   C2  2183
M35   C3   C4  2239
M36   C3   C10  2225
M37   C3   C11  2091
M38   C3   C12  2212
M39   C3   C14  2574
M40   C3   C15  2385
M41   C3   C16  2621
M42   C4   C1  2018
M43   C4   C2  2795
M44   C4   C3  2650
M45   C4   C5  2440
M46   C4   C8  2022
M47   C4   C9  2251
M48   C4   C10  2080
M49   C4   C11  2833
M50   C4   C12  2603
M51   C4   C13  2614
M52   C4   C14  2054
M53   C4   C15  2547
M54   C4   C16  2035
M55   C4   C17  2911
M56   C5   C0  2136
M57   C5   C1  2914
M58   C5   C3  2810
M59   C5   C6  2530
M60   C5   C7  2635
M61   C5   C9  2502
M62   C5   C10  2300
M63   C5   C12  2319
M64   C5   C13  2930
M65   C5   C14  2677
M66   C5   C15  2027
M67   C5   C16  2041
M68   C5   C17  2940
M69   C6   C1  2799
M70   C6   C5  2039
M71   C6   C7  2053
M72   C6   C9  2434
M73   C6   C10  2622
M74   C6   C11  2946
M75   C6   C12  2286
M76   C6   C15  2939
M77   C6   C17  2723
M78   C7   C0  2256
M79   C7   C1  2489
M80   C7   C2  2756
M81   C7   C4  2758
M82   C7   C5  2482
M83   C7   C6  2941
M84   C7   C9  2516
M85   C7   C10  2334
M86   C7   C12  2024
M87   C7   C13  2398
M88   C7   C17  2142
M89   C8   C0  2519
M90   C8   C1  2545
M91   C8   C3  2650
M92   C8   C4  2787
M93   C8   C5  2975
M94   C8   C6  2309
M95   C8   C9  2586
M96   C8   C13  2155
M97   C8   C15  2962
M98   C8   C16  2282
M99   C8   C17  2773
M100   C9   C0  2179
M101   C9   C3  2330
M102   C9   C4  2445
M103   C9   C5  2751
M104   C9   C7  2578
M105   C9   C10  2397
M106   C9   C11  2153
M107   C9   C12  2010
M108   C9   C14  2956
M109   C9   C15  2481
M110   C9   C16  2253
M111   C10   C0  2612
M112   C10   C1  2135
M113   C10   C2  2365
M114   C10   C3  2966
M115   C10   C4  2378
M116   C10   C6  2600
M117   C10   C8  2184
M118   C10   C12  2388
M119   C10   C14  2006
M120   C10   C15  2954
M121   C11   C0  2634
M122   C11   C1  2188
M123   C11   C2  2300
M124   C11   C3  2115
M125   C11   C4  2427
M126   C11   C5  2252
M127   C11   C7  2440
M128   C11   C8  2511
M129   C11   C9  2348
M130   C11   C10  2250
M131   C11   C12  2055
M132   C11   C13  2248
M133   C11   C14  2789
M134   C11   C15  2479
M135   C11   C16  2061
M136   C11   C17  2854
M137   C12   C1  2995
M138   C12   C4  2184
M139   C12   C5  2864
M140   C12   C6  2933
M141   C12   C7  2545
M142   C12   C8  2815
M143   C12   C9  2595
M144   C12   C10  2982
M145   C12   C11  2526
M146   C12   C13  2900
M147   C12   C14  2084
M148   C12   C15  2711
M149   C12   C16  2518
M150   C12   C17  2911
M151   C13   C2  2901
M152   C13   C4  2470
M153   C13   C8  2906
M154   C13   C10  2945
M155   C13   C12  2405
M156   C13   C14  2531
M157   C13   C15  2241
M158   C13   C16  2736
M159   C13   C17  2204
M160   C14   C0  2013
M161   C14   C2  2099
M162   C14   C3  2371
M163   C14   C5  2351
M164   C14   C6  2180
M165   C14   C7  2561
M166   C14   C8  2925
M167   C14   C9  2750
M168   C14   C10  2963
M169   C14   C11  2341
M170   C14   C12  2391
M171   C14   C13  2160
M172   C14   C15  2265
M173   C14   C16  2125
M174   C15   C1  2211
M175   C15   C2  2061
M176   C15   C3  2279
M177   C15   C4  2824
M178   C15   C5  2358
M179   C15   C6  2628
M180   C15   C8  2477
M181   C15   C9  2156
M182   C15   C10  2129
M183   C15   C12  2708
M184   C15   C13  2015
M185   C15   C16  2174
M186   C15   C17  2652
M187   C16   C0  2217
M188   C16   C1  2671
M189   C16   C2  2139
M190   C16   C3  2093
M191   C16   C4  2989
M192   C16   C6  2869
M193   C16   C8  2691
M194   C16   C9  2473
M195   C16   C12  2624
M196   C16   C13  2096
M197   C16   C14  2982
M198   C16   C15  2916
M199   C17   C0  2255
M200   C17   C1  2681
M201   C17   C2  2763
M202   C17   C6  2663
M203   C17   C7  2346
M204   C17   C8  2817
M205   C17   C10  2382
M206   C17   C12  2044
M207   C17   C14  2956
M208   C17   C15  2120
M209   C17   C16  2218
