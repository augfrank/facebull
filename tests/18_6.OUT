38277
7 10 24 37 46 67 70 86 96 100 112 130 147 157 161 181 190 203
